### ISSUE TITLE
[Idea/FeatureRequest]: Continuously location sharing

### ISSUE THREAD

**Issue by dennisguse** [siacs/Conversations#3163]
What?
---
This might be specific use case, but sometimes I would like to share my location continuously via Conversations.

Why?
---
If I meet with other people, it is sometimes beneficial (especially when late) to share my location continuously.
For example, while riding a bicycle or driving a car.

Current solution
---
_Manually_ share location as often as needed/possible.
Touching the phone while driving, however, is often not allowed...

Desired solution
---
So, basically I want to share my location with contact(s) on my chat rooster for some time.
Turning the functionality on while I need it and turn it off manually later.
For me, it would be sufficient to have just _one_ continuous location sharing concurrently.

**Comment by dennisguse** [comment]
Depends. 
I just want to sent the location continuously to a contact while not necessarily requiring/allowing him to subscribe to the updates. 

Simplest implementation would be to really just trigger sending the location at a certain period. 

On September 1, 2018 7:04:02 PM UTC, Licaon_Kter <notifications@github.com> wrote:
>/close as duplicate #1069 ?
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/siacs/Conversations/issues/3163#issuecomment-417880512

-- 
Dennis Guse

**Comment by therob84** [comment]
Are there any news on the ability to share the location as interval (coming now from Conversations 2.5.8+fcr), e.g. sending location once a minute for 15 minutes?

**Comment by blaueente** [comment]
Note that osmand has telegram integration: 
https://osmand.net/blog/osmand-telegram-released 
https://github.com/osmandapp/Osmand/tree/master/OsmAnd-telegram

Hilarious:
> We have chosen Telegram as the most open social platform since it has an Open API, Open SDK and, moreover, it will eventually have an Open Server implementation (Blockchain).

I think conversations would have been a much better match, as it is actually much more open.
Anyway, maybe it might be possible to take the osmand-telegram app as a basis to quickly get osmand integration?

**Comment by licaon-kter** [comment]
In the mean time I recommend https://github.com/bilde2910/Hauk ;)

**Comment by xmgz** [comment]
thanks @licaon-kter 
but one would need a web server (or access to one) where to install hauk's backend.

Conversations would be perfect.

My user case would be as security beacon when cycling (any outdoors activity). Setting a time frame (ex. 6 hours) within you could share location AUTOMATICALLY every X minutes (ex. 10) with any user OR private room to tell your family you're safe and riding. Give GPS access and share location with that room/user without user interaction, simply the geo:longitude,latitude link 

Many other user experiences could be covered.


Please consider it, thanks anyway
